src/ctapipe/image/extractor.py

```diff
@@ -1295,6 +1296,21 @@ def __call__(
         )
 
 
+class VarianceExtractor(ImageExtractor):
+
+    """
+    Extractor that calculates
```

Comment:
First line needs to be a single line summary followed by an empty line, then you can give more details.

I would propose:

```
Compute variance over samples for each waveform.
```